src/backoffice/core/property-editors/uis/label/property-editor-ui-label.element.ts

```diff
@@ -1,8 +1,11 @@
-import { html } from 'lit';
-import { customElement, property } from 'lit/decorators.js';
+import { html, nothing } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
 import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
 import { UmbPropertyEditorExtensionElement } from '@umbraco-cms/backoffice/extensions-registry';
 import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import type { DataTypePropertyPresentationModel } from '@umbraco-cms/backoffice/backend-api';
+
+type umbracoDataValueType = 'STRING' | 'DECIMAL' | 'DATE/TIME' | 'TIME' | 'INTEGER' | 'BIG INTEGER' | 'LONG STRING';
```

Comment:
Is this saved on the config model anywhere? Can we ensure some more type safety somehow, e.g. can they use the same type variable?

Reply:
Its not from what I can see.
Im not sure if these are needed in the property editor UI either, as it doesn't seem like label even has any inputs in the first place?
The old backoffice has all these different label property editors with the different types but Im not sure if they really do anyting in the frontend at all.

Reply:
Looks like it is used to tell the server database what the column type should be. These types should be supported, but not sure if they are stringified like you do here: 
<img width="977" alt="image" src="https://user-images.githubusercontent.com/752371/235893200-63531821-0817-4f20-b8ce-a8fa3ca45133.png">

Reply:
We don't need to know any of this in the UI. It is used only in the configuration of the underlying property editor but has no effect on the UI.